backends/p4tools/testgen/test/small-step/reachability.cpp

```diff
@@ -57,6 +58,10 @@ ReturnedInfo loadExampleForReachability(const char* curFile) {
     options.file = sourcePath;
     options.file += "testdata/p4_16_samples/";
     options.file += curFile;
+    if (access(options.file, F_OK) != 0) {
```

Comment:
What happens is this is not true. The test should not pass.

Reply:
I made refactoring here. The problem was that the get two sources of the test: `testdata/p4_16_samples` and from `p4tools/testgen/targets/bmv2/p4-programs`. Now `curFile` should contain relative path from p4c folder and checking is not needed.